packages/dds/test-dds-utils/src/ddsFuzzHarness.ts

```diff
@@ -184,6 +195,34 @@ function getSaveInfo(
 	return { saveOnFailure: true, filepath };
 }
 
+export class DDSFuzzHandle<T = IFluidLoadable & FluidObject> implements IFluidHandle {
+	readonly absolutePath: string;
+
+	public get IFluidHandle(): DDSFuzzHandle<T> {
+		return this;
+	}
+
+	public get isAttached(): boolean {
+		throw new Error("Method not implemented.");
+	}
+
+	// eslint-disable-next-line @typescript-eslint/no-explicit-any
+	public async get(): Promise<any> { // i know this isn't right
```

Comment:
Same issue as above - the type here shouldn't be any, but I'm not sure what type would fulfill the requirements.